.ci/release.sh

```diff
@@ -11,7 +11,8 @@ go get github.com/github-release/github-release
 github-release release \
     --user ${USER} \
     --repo ${REPO} \
-    --tag "${TAG_NAME}"
+    --tag "${TAG_NAME}" \
+    --description "Please check the changelog - https://www.elastic.co/guide/en/apm/agent/php/current/release-notes.html"
```

Comment:
Is this the text that goes on GitHub? Here?

<img width="1353" alt="Screen Shot 2020-10-12 at 9 00 33 AM" src="https://user-images.githubusercontent.com/5618806/95767344-76d98b80-0c69-11eb-82a6-a5d65ef50b3b.png">

If so, should we change this to a markdown link so that users can just click on it? Something like:
```suggestion
    --description "For more information, please see the [changelog](https://www.elastic.co/guide/en/apm/agent/php/current/release-notes.html)."
```

Reply:
That's brilliant :)